client/state/data-getters/index.js

```diff
@@ -157,6 +157,20 @@ export const requestGutenbergDraftPost = ( siteId, draftId ) =>
 		{ fromApi: () => data => [ [ draftId, data ] ] }
 	);
 
+export const requestGutenbergDemoContent = draftId =>
```

Comment:
why does this require a draft id? isn't it the same the same the same all the time?

Reply:
To be honest I didn't pay much attention at that part, but more at making the content show up.
At a quick glance, every other getter used a unique ID (at a less quick glance it's not true, I know 🙂), so I just went and added it as well.